src/modules/administrator-guide/pages/roles.adoc

```diff
@@ -96,3 +96,14 @@ Instead of deleting roles, you can <<archiving-roles,archive the role>>.
 . click and confirm the btn:[delete] button.
 
 image::role-edit.png[role="data-zoomable"]
+
+== Cloning role permissions
+
+.To clone permission of role to a specific roles:
+. Navigate to menu:system[roles],
+. click on the btn:[clone permissions] button,
+. select the role(s) to clone permissions to,
+. press the btn:[clone] button.
+
+image::role-clone-permissions-button.png[role="data-zoomable"]
+image::role-clone-permissions.png[role="data-zoomable"]
```

Comment:
Add a caption to both of the images:

```
.Caption goes here; describe what the image shows; end with a period (.)
image::blahblah.png[]
```

If you are feeling it, you can also annotate the images (not a requirement for now) https://docs.cortezaproject.org/corteza-docs/2021.9/developer-guide/documentation/annotations.html